api/python/quilt3/packages.py

```diff
@@ -1462,7 +1462,7 @@ def check_latest_hash():
         def physical_key_is_temp_file(pk):
             if not pk.is_local():
                 return False
-            return pathlib.Path(pk.path).parent == APP_DIR_TEMPFILE_DIR
+            return pathlib.Path(pk.path).parent.resolve() == APP_DIR_TEMPFILE_DIR.resolve()
```

Comment:
Does it make sense to do this normalization when we create `PhysicalKey` instances?

Reply:
I'd agree with both of those points.  In theory, the resolution should not have a problematic effect -- but knowing definitively is another matter.